### ISSUE TITLE
Proposal to create RC2 of v0.10.0

### ISSUE THREAD

**Issue by hdamker** [camaraproject/QualityOnDemand#260]
**Problem description**

Since the creation of the first release candidate of v0.10.0 there were several issues relevant for v0.10.0 raised and PRs prepared which are now ready to be merged. Due do the amount of (little) changes I recommend to create a second release candidate.

**Expected action**

The following PRs are (almost) ready to be merged:

* https://github.com/camaraproject/QualityOnDemand/pull/259 ... this PR should be merged first as it includes the (temporary) change of the version number back to v0.10.0-wip (read as: "unreleased").
* https://github.com/camaraproject/QualityOnDemand/pull/258
* https://github.com/camaraproject/QualityOnDemand/pull/256
* https://github.com/camaraproject/QualityOnDemand/pull/252

Optional (if agreed):

* https://github.com/camaraproject/QualityOnDemand/pull/255

Plus a new PR which will document the (incremental) changes between RC1 and RC2 within CHANGELOG, update the README and set the version to 0.10.0-rc2

**Additional context**
